README.md

```diff
@@ -1,26 +1,19 @@
-# Build Switch Images - buildimage
+## Build Switch Images - buildimage
 
-# Description
-Build an [Open Network Install Environment (ONIE)](https://github.com/opencomputeproject/onie) compatiable network operating system (NOS) installer image for network switches, and also build docker images running inside the NOS.
+Build an [Open Network Install Environment (ONIE)](https://github.com/opencomputeproject/onie) compatible network operating system (NOS) installer image for network switches, and also build docker images running inside the NOS.
 
-# Prerequisite
-## 1. Build environment
-Preferably use [the Dockerfile](https://github.com/Azure/sonic-build-tools/blob/master/sonic-slave/Dockerfile), or use Debian Jessie and manually install packages appearing in the Dockerfile.
-## 2. Linux kernel with switch drivers
-Build the [Azure/sonic-linux-kernel](https://github.com/Azure/sonic-linux-kernel) project and copy the output .deb file into ./deps directory.
-
-## 3. initramfs-tools with loop device support
-Run the script to build the .deb file into ./deps directory.
+## Prerequisites
+**1.** Build environment - Preferably use [the Dockerfile](https://github.com/Azure/sonic-build-tools/blob/master/sonic-slave/Dockerfile), or use Debian Jessie and manually install packages appearing in the Dockerfile.
```

Comment:
Dell - Internal Use - Confidential
Added a space after 1.

From: lguohan [mailto:notifications@github.com]
Sent: Tuesday, August 02, 2016 4:09 PM
To: Azure/sonic-buildimage <sonic-buildimage@noreply.github.com>
Cc: Buck, Amy <Amy_Buck@Dell.com>; Author <author@noreply.github.com>
Subject: Re: [Azure/sonic-buildimage] Update README.md (#2)


In README.md<https://github.com/Azure/sonic-buildimage/pull/2#discussion_r73254981>:

>

> -# Prerequisite

> -## 1. Build environment

> -Preferably use [the Dockerfile](https://github.com/Azure/sonic-build-tools/blob/master/sonic-slave/Dockerfile), or use Debian Jessie and manually install packages appearing in the Dockerfile.

> -## 2. Linux kernel with switch drivers

> -Build the [Azure/sonic-linux-kernel](https://github.com/Azure/sonic-linux-kernel) project and copy the output .deb file into ./deps directory.

> -

> -## 3. initramfs-tools with loop device support

> -Run the script to build the .deb file into ./deps directory.

> +## Prerequisites

> +**1.** Build environment - Preferably use [the Dockerfile](https://github.com/Azure/sonic-build-tools/blob/master/sonic-slave/Dockerfile), or use Debian Jessie and manually install packages appearing in the Dockerfile.

this 1. does not format well in github.com, can you check.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/sonic-buildimage/pull/2/files/3bc2f7b051a939154b4b94bb651780d30325acd4#r73254981>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AToLHI161lIspRY3Yx6wmTD9igYJObMCks5qb84ggaJpZM4Ja18Y>.